docs/docs/concepts/langgraph_studio.md

```diff
@@ -0,0 +1,33 @@
+# LangGraph Studio
```

Comment:
I don't know that the overview is motivating enough here.

How exactly does it improve collaboration with teammates?
Why is an IDE useful?